pmm-app/tests/pages/pmmInventoryPage.js

```diff
@@ -16,6 +16,11 @@ module.exports = {
     nodesLinkOld: '//a[contains(text(), "Nodes")]',
     pmmAgentLocator: '//table//td[contains(text(), "PMM Agent")]',
     serviceIdLocatorPrefix: '//table//tr/td[4][contains(text(),"',
+    deleteButton: '//span[contains(text(), "Delete")]',
+    proceedButton: '//span[contains(text(), "Proceed")]',
+    forceModeCheckbox:
+      '//div[@data-qa="form-field-force"]/descendant::span[@class="checkbox-container__checkmark"]',
```

Comment:
this can be simpler `div[data-qa="form-field-force"] span.checkbox-container__checkmark`

Reply:
Same as above :)

Reply:
Our decision was to use xpath only when we can't use css, IIRC